src/GraphQL/Utilities/Federation/AnyValue.cs

```diff
@@ -1,10 +1,12 @@
+#nullable enable
+
 using GraphQL.Language.AST;
 
 namespace GraphQL.Utilities.Federation
 {
     public class AnyValue : ValueNode<object>
     {
-        public AnyValue(object value) : base(value)
+        public AnyValue(object? value) : base(value!)
```

Comment:
)

Reply:
Maybe we can change the base type to `ValueNode<object?>` which could be more appropriate. I really don’t know how AnyValue is used so I was probably trying to make as few changes as possible.